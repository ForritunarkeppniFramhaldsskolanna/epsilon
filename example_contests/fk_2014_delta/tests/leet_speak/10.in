NFXmqsPsO0xNz xauv5XacfhYSinXkMnupk5z
