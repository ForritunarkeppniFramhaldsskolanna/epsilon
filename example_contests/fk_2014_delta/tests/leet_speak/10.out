|\||=><mqs|Ds00x|\|z xauv5><acfh`/5in><k|\/|nupk5z
